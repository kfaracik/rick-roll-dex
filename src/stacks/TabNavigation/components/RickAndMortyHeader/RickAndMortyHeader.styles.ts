import {StyleSheet} from 'react-native';

export const styles = StyleSheet.create({
  headerImage: {
    width: 105,
    height: 32,
  },
});
