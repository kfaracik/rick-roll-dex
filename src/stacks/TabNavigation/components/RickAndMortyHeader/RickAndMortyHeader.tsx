import {Image} from 'react-native';
import React from 'react';
import rickAndMorty from '../../../../shared/assets/images/rickAndMorty/rickAndMorty.webp';
import {styles} from './RickAndMortyHeader.styles';

export const RickAndMortyHeader = () => {
  return (
    <Image
      resizeMode="contain"
      source={rickAndMorty}
      style={styles.headerImage}
    />
  );
};
