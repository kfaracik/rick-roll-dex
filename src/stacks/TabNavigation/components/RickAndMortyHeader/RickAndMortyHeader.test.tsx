import React from 'react';
import {render} from '@testing-library/react-native';
import {RickAndMortyHeader} from './RickAndMortyHeader';

describe('RickAndMortyHeader component', () => {
  it('should generate snapshot', () => {
    const {toJSON} = render(<RickAndMortyHeader />);
    expect(toJSON()).toMatchSnapshot();
  });
});
