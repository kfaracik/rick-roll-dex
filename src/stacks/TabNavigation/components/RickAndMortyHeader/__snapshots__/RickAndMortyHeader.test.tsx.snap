// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should generate snapshot 1`] = `
<Image
  resizeMode="contain"
  source={
    {
      "testUri": "../../../src/shared/assets/images/rickAndMorty/rickAndMorty.webp",
    }
  }
  style={
    {
      "height": 32,
      "width": 105,
    }
  }
/>
`;

exports[`RickAndMortyHeader component should generate snapshot 1`] = `
<Image
  resizeMode="contain"
  source={
    {
      "testUri": "../../../src/shared/assets/images/rickAndMorty/rickAndMorty.webp",
    }
  }
  style={
    {
      "height": 32,
      "width": 105,
    }
  }
/>
`;
